src/main/java/frc/utils/ToleranceUtils.java

```diff
@@ -0,0 +1,21 @@
+package frc.utils;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.robot.constants.MathConstants;
+
+public class ToleranceUtils {
+
+	public static boolean wrappedIsNear(Rotation2d tolerance, Rotation2d rotation, Rotation2d wantedRotation) {
+		return Math.abs(wantedRotation.getRadians() - rotation.getRadians()) <= tolerance.getRadians()
+			|| Math.abs(wantedRotation.getRadians() - rotation.getRadians()) >= MathConstants.FULL_CIRCLE.getRadians() - tolerance.getRadians();
```

Comment:
It will be easier to read if u switch the order of the members of the equation:
```suggestion
			||MathConstants.FULL_CIRCLE.getRadians() - tolerance.getRadians() <= Math.abs(wantedRotation.getRadians() - rotation.getRadians())
```